cmd/kubectl-k8ssandra/register/command.go

```diff
@@ -0,0 +1,72 @@
+package register
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var RegisterClusterCmd = &cobra.Command{
+	Use:   "register [flags]",
+	Short: "register a data plane into the control plane.",
+	Long:  `register creates a ServiceAccount on a source cluster, copies its credentials and then creates a secret containing them on the destination cluster. It then also creates a ClientConfig on the destination cluster to reference the secret.`,
+	Run:   entrypoint,
+}
+
+func Init(cmd *cobra.Command, streams genericclioptions.IOStreams) {
+	RegisterClusterCmd.Flags().String("source-kubeconfig",
+		"",
+		"path to source cluster's kubeconfig file - defaults to KUBECONFIG then ~/.kube/config")
+	RegisterClusterCmd.Flags().String("dest-kubeconfig",
+		"",
+		"path to destination cluster's kubeconfig file - defaults to KUBECONFIG then ~/.kube/config")
+	RegisterClusterCmd.Flags().String("source-context", "", "context name for source cluster")
+	RegisterClusterCmd.Flags().String("dest-context", "", "context name for destination cluster")
+	RegisterClusterCmd.Flags().String("source-namespace", "k8ssandra-operator", "namespace containing service account for source cluster")
+	RegisterClusterCmd.Flags().String("dest-namespace", "k8ssandra-operator", "namespace where secret and clientConfig will be created on destination cluster")
+	RegisterClusterCmd.Flags().String("serviceaccount-name", "k8ssandra-operator", "serviceaccount name for destination cluster")
+	RegisterClusterCmd.Flags().String("destination-name", "remote-k8ssandra-operator", "name for remote clientConfig and secret on destination cluster")
+
+	if err := RegisterClusterCmd.MarkFlagRequired("source-context"); err != nil {
+		panic(err)
+
+	}
+	if err := RegisterClusterCmd.MarkFlagRequired("dest-context"); err != nil {
+		panic(err)
+	}
+	cmd.AddCommand(RegisterClusterCmd)
+}
+
+func entrypoint(cmd *cobra.Command, args []string) {
+	executor := NewRegistrationExecutorFromRegisterClusterCmd(*cmd)
+	for i := 0; i < 30; i++ {
+		res := executor.RegisterCluster()
+		switch {
+		case res.IsError():
+			fmt.Println("Registration continuing", res.GetError())
```

Comment:
issue: There's already logging library in k8ssandra-client, please use it. It provides the ability to keep consistent customizable output (and time etc).

Reply:
Hmmmm! Acknowledging that the [library](https://github.com/charmbracelet/log) you've chosen here has pretty colours, I'm not sure that this is one of the big standard ones I see used! 

I'm also not sure it was in mcctl, do you want to use it there too? For my future reference...

I'm not sure I like this technology choice but let's go with it. I'll implement this feedback.

Reply:
charmbracelet is one of the most popular (if not the most popular) command line / TUI libraries in Go. mcctl has something similar as well as intention to move to this one also (because the one in mcctl has less features or maintenance).